src/config.py

```diff
@@ -0,0 +1,217 @@
+
+    slack_channel = os.getenv("SLACK_CHANNEL", None)
+    slack_token = os.getenv("SLACK_TOKEN", None)
+
+    node_url = f"https://{network}.infura.io/v3/{infura_key}"
+
+    #
+    payment_safe_address = Web3.to_checksum_address(
+        os.environ.get("SAFE_ADDRESS", "0xA03be496e67Ec29bC62F01a428683D7F9c204930")
+    )
+    short_name = {
+        "mainnet": "eth",
+        "rinkeby": "rin",
+        "gnosis": "gno",
+        "goerli": "gor",
+    }[network]
+    csv_app_hash = "Qme49gESuwpSvwANmEqo34yfCkzyQehooJ5yL7aHmKJnpZ"
+    safe_url = f"https://app.safe.global/{short_name}:{payment_safe_address}"
+    airdrop_url = (
+        f"{safe_url}/apps?appUrl=https://cloudflare-ipfs.com/ipfs/{csv_app_hash}/"
+    )
+    safe_queue_url = f"{safe_url}/transactions/queue"
+
+    result = AccountingConfig(
+        payment_config=PaymentConfig(
+            network=EthereumNetwork.MAINNET,
+            cow_token_address=Address("0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB"),
+            payment_safe_address=Web3.to_checksum_address(
+                "0xA03be496e67Ec29bC62F01a428683D7F9c204930"
+            ),
+            signing_key=signing_key,
+            safe_queue_url=safe_queue_url,
+            csv_airdrop_url=airdrop_url,
+            verification_docs_url=docs_url,
+            weth_address=Web3.to_checksum_address(
+                "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
+            ),
+        ),
+        orderbook_config=OrderbookConfig(
+            prod_db_url=prod_db_url,
+            barn_db_url=barn_db_url,
+        ),
+        dune_config=DuneConfig(dune_api_key=dune_api_key, dune_blockchain="ethereum"),
+        node_config=NodeConfig(node_url=node_url),
+        reward_config=RewardConfig(
+            reward_token_address=Address("0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB"),
+            batch_reward_cap_upper=12 * 10**15,
+            batch_reward_cap_lower=10 * 10**15,
+            quote_reward_cow=6 * 10**18,
+            quote_reward_cap_native=6 * 10**14,
+            service_fee_factor=Fraction(15, 100),
+            cow_bonding_pool=Address("0x5d4020b9261f01b6f8a45db929704b0ad6f5e9e6"),
+        ),
+        protocol_fee_config=ProtocolFeeConfig(
+            protocol_fee_safe=Address("0xB64963f95215FDe6510657e719bd832BB8bb941B"),
+            partner_fee_cut=0.15,
+            partner_fee_reduced_cut=0.10,
+            reduced_cut_address="0x63695Eee2c3141BDE314C5a6f89B98E62808d716",
+        ),
+        buffer_accounting_config=BufferAccountingConfig(include_slippage=True),
+        io_config=IOConfig(
+            project_root_dir=project_root_dir,
+            log_config_file=log_config_file,
+            query_dir=query_dir,
+            csv_output_dir=file_out_dir,
+            dashboard_dir=dashboard_dir,
+            slack_channel=slack_channel,
+            slack_token=slack_token,
+        ),
+    )
+
+    return result
+
+
+config = get_accounting_config(os.environ.get("NETWORK", "mainnet"))
```

Comment:
I think it's best to set as an enum, just so we can verify network with `NETWORK.is_mainnet()` rather than `network == "mainnet"`

Reply:
I added a `Network` enum class.

What is the use case for a function like `is_mainnet`? I am using match clauses for the setup at the moment.

Reply:
I think your approach to use match cases is better actually. I didn't think of doing it like this, using a method like `is_mainnet` is a habit I got into before match cases were introduced to avoid magic strings and make this sort of logic more maintainable. I don't think there's a material difference in functionality, but this looks much more concise.

I think technically match cases are preferred when we want to check multiple different states (such as this where chain can be arbitrum, gnosis, or mainnet). And using a method such as `is_mainnet` would be preferred if there's only one attribute to check with one if statement (such as only performing a certain step on mainnet). I'll make this change on the dune-sync config as well to keep things consistent.